### ISSUE TITLE
Flexslider resize not working properly

### ISSUE THREAD

**Issue by hgnl96** [woocommerce/FlexSlider#1493]
I use Flexslider on my website. But when I resize my screen or rotate my mobile phone, Flexslider is do not resize the slider properly.

I saw the docs from Woothemes, with the grid function, but it doesn't work for 100%. Does someone know why?

My flexslider function looks like this:

```
    loadFlexsliderCarousel = function(){
        var $window = $(window),
        flexslider = { vars:{} };

        function getGridSize() {
        return (window.innerWidth < 600) ? 1 :
               (window.innerWidth < 768) ? 2 :
               (window.innerWidth < 1024) ? 3 : 4; 
        };

        $('.logos .flexslider').flexslider({
            animation: "slide",
            animationLoop: false,
            itemWidth: 200,
            itemMargin: 20,
            directionNav: true,
            controlNav: false,
            minItems: getGridSize(),
            maxItems: getGridSize(),
            start: function(slider){
                flexslider = slider;
            }
        });

        $window.resize(function() {
            var gridSize = getGridSize();

            flexslider.vars.minItems = gridSize;
            flexslider.vars.maxItems = gridSize;
        });     
    };

```

Now I use this function, but this is not working, my browser is crashing because the 0.5 second on the setTimeout.

```
    $(window).load(function() {
        loadFlexsliderCarousel();
    });

    $(window).bind('orientationchange', function() {
        var slider = $('.logos .flexslider').data('flexslider');
        slider.flexAnimate(0, true, true);
    });

    $(window).bind('resize', function() { 
        setTimeout(function(){ 
            var slider = $('.logos  .flexslider').data('flexslider');   
            slider.resize();
        }, 500);
    });
```

Any ideas?

**Comment by inier** [comment]
Yes, I have this problem too.

**Comment by laobie** [comment]
Me, too.

**Comment by nmklong** [comment]
still having this problem, it's not solved yet? when container is width 100% then it will set the li to >600px no matter how i shrink the viewport's width

**Comment by hgnl96** [comment]
I fix it by using Slick slider, it's a easy library and works great!

Source: http://kenwheeler.github.io/slick/

**Comment by juergen-venne** [comment]
I changed the binding of the resize method. That fixed this issue for me.
https://github.com/woocommerce/FlexSlider/issues/1690

**Comment by jeffikus** [comment]
Hi there, I released 2.7.0 last week, please let me know if the issue persists after updating.
